bpfd/src/bpf.rs

```diff
@@ -5,19 +5,18 @@ use std::{collections::HashMap, convert::TryInto, fs};
 
 use anyhow::anyhow;
 use aya::{programs::TracePoint, BpfLoader};
-use bpfd_api::{config::Config, util::directories::*};
-use log::{debug, info};
+use bpfd_api::{config::Config, util::directories::*, ProgramType, TcProceedOn};
```

Comment:
Wait a second... In bpf.rs we're internal right? Shouldn't we only be using internal types here (i.e those mirrored in command.rs)